WordPress/Tests/Commenting/HooksInlineDocsUnitTest.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+// phpcs:set WordPress.Commenting.HooksInlineDocs allowed_extra_versions[] MU
```

Comment:
This doesn't have to be specified at the top - you could move it to just before it's needed, so that you can check that the behaviour works correctly without any allowed_extra_versions having been set.

Reply:
b2855e67